WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -2986,6 +2986,14 @@ function(){
    this.parent._editDelegateButton();
 }
 
+//overriding no results container getter function for custom behavior
+ZmAccountDelegatesListView.prototype._getNoResultsContainer =
+function() {
+	var div = document.createElement("div");
+	div.className = "ZmListViewTableNoResultsContainer";
```

Comment:
trying to understand why we need to override and add extra class for no results container
as it can be targetted using css selector "div .NoResults" also